frame/support/src/lib.rs

```diff
@@ -1930,12 +1915,12 @@ pub mod pallet_prelude {
 /// 	- `#[compact]` must now be written `#[pallet::compact]`
 /// 	- `#[weight = ..]` must now be written `#[pallet::weight(..)]`
 ///
-/// 8. **migrate event**:
+/// 7. **migrate event**:
 /// 	rewrite as a simple enum under with the attribute `#[pallet::event]`,
 /// 	use `#[pallet::generate_deposit($vis fn deposit_event)]` to generate deposit_event,
 /// 	use `#[pallet::metadata(...)]` to configure the metadata for types in order not to break them.
-/// 9. **migrate error**: just rewrite it with attribute `#[pallet::error]`.
-/// 10. **migrate storage**:
+/// 8. **migrate error**: just rewrite it with attribute `#[pallet::error]`.
```

Comment:
```suggestion
/// 8. **migrate error**: rewrite it with attribute `#[pallet::error]`.
```

Generally I'd avoid using words like "just" or "simply" – for an inexperienced reader that struggles to follow along it can sound condescending. What is "just" for you or me can be a "crying in despair for hours" for someone else. :)

Reply:
yes I agree

Reply:
done, I removed other "just"